### ISSUE TITLE
Network Boot enabled by default on OptiPlex 7010/9010 v0.1.0-rc1

### ISSUE THREAD

**Issue by filipleple** [Dasharo/dasharo-issues#979]
### Component

Dasharo firmware

### Device

Dell OptiPlex 7010, Dell OptiPlex 9010

### Dasharo version

v0.1.0-rc1

### Brief summary

Network Boot is enabled by default, unlike in most other Dasharo FW releases

### How reproducible

100%

### How to reproduce

Reset to defaults, observe that Network Boot is set  to enabled

### Expected behavior

It should default to disabled

### Actual behavior

It defaults to enabled

**Comment by pietrushnic** [comment]
@BeataZdunczyk, why was this issue not reviewed for so long? Does it block release?

**Comment by BeataZdunczyk** [comment]
@pietrushnic 

>  Does it block release?

No, it does not. It will be marked as a known issue in this initial release. Moved to the next milestone: https://github.com/Dasharo/dasharo-issues/milestone/49
